chirp/drivers/uvk5.py

```diff
@@ -0,0 +1,2032 @@
+STEPS = [2.5, 5.0, 6.25, 10.0, 12.5, 25.0, 8.33]
+
+# ctcss/dcs codes
+TMODES = ["", "Tone", "DTCS", "DTCS"]
+TONE_NONE = 0
+TONE_CTCSS = 1
+TONE_DCS = 2
+TONE_RDCS = 3
+
+
+CTCSS_TONES = [
+    67.0, 69.3, 71.9, 74.4, 77.0, 79.7, 82.5, 85.4,
+    88.5, 91.5, 94.8, 97.4, 100.0, 103.5, 107.2, 110.9,
+    114.8, 118.8, 123.0, 127.3, 131.8, 136.5, 141.3, 146.2,
+    151.4, 156.7, 159.8, 162.2, 165.5, 167.9, 171.3, 173.8,
+    177.3, 179.9, 183.5, 186.2, 189.9, 192.8, 196.6, 199.5,
+    203.5, 206.5, 210.7, 218.1, 225.7, 229.1, 233.6, 241.8,
+    250.3, 254.1
+]
+
+# lifted from ft4.py
+DTCS_CODES = [
+    23,  25,  26,  31,  32,  36,  43,  47,  51,  53,  54,
+    65,  71,  72,  73,  74,  114, 115, 116, 122, 125, 131,
+    132, 134, 143, 145, 152, 155, 156, 162, 165, 172, 174,
+    205, 212, 223, 225, 226, 243, 244, 245, 246, 251, 252,
+    255, 261, 263, 265, 266, 271, 274, 306, 311, 315, 325,
+    331, 332, 343, 346, 351, 356, 364, 365, 371, 411, 412,
+    413, 423, 431, 432, 445, 446, 452, 454, 455, 462, 464,
+    465, 466, 503, 506, 516, 523, 526, 532, 546, 565, 606,
+    612, 624, 627, 631, 632, 654, 662, 664, 703, 712, 723,
+    731, 732, 734, 743, 754
+]
+
+FLOCK_LIST = ["Off", "FCC", "CE", "GB", "430", "438"]
+
+SCANRESUME_LIST = ["TO: Resume after 5 seconds",
+                   "CO: Resume after signal dissapears",
+                   "SE: Stop scanning after receiving a signal"]
+
+WELCOME_LIST = ["Full Screen", "Welcome Info", "Voltage"]
+KEYPADTONE_LIST = ["Off", "Chinese", "English"]
+LANGUAGE_LIST = ["Chinese", "English"]
+ALARMMODE_LIST = ["SITE", "TONE"]
+REMENDOFTALK_LIST = ["Off", "ROGER", "MDC"]
+RTE_LIST = ["Off", "100ms", "200ms", "300ms", "400ms",
+            "500ms", "600ms", "700ms", "800ms", "900ms"]
+
+MEM_SIZE = 0x2000  # size of all memory
+PROG_SIZE = 0x1d00  # size of the memory that we will write
+MEM_BLOCK = 0x80  # largest block of memory that we can reliably write
+
+# fm radio supported frequencies
+FMMIN = 76.0
+FMMAX = 108.0
+
+# bands supported by the UV-K5
+BANDS = {
+        0: [50.0, 76.0],
+        1: [108.0, 135.9999],
+        2: [136.0, 199.9990],
+        3: [200.0, 299.9999],
+        4: [350.0, 399.9999],
+        5: [400.0, 469.9999],
+        6: [470.0, 600.0]
+        }
+
+# for radios with modified firmware:
+BANDS_NOLIMITS = {
+        0: [18.0, 76.0],
+        1: [108.0, 135.9999],
+        2: [136.0, 199.9990],
+        3: [200.0, 299.9999],
+        4: [350.0, 399.9999],
+        5: [400.0, 469.9999],
+        6: [470.0, 1300.0]
+        }
+
+SPECIALS = {
+        "F1(50M-76M)A": 200,
+        "F1(50M-76M)B": 201,
+        "F2(108M-136M)A": 202,
+        "F2(108M-136M)B": 203,
+        "F3(136M-174M)A": 204,
+        "F3(136M-174M)B": 205,
+        "F4(174M-350M)A": 206,
+        "F4(174M-350M)B": 207,
+        "F5(350M-400M)A": 208,
+        "F5(350M-400M)B": 209,
+        "F6(400M-470M)A": 210,
+        "F6(400M-470M)B": 211,
+        "F7(470M-600M)A": 212,
+        "F7(470M-600M)B": 213
```

Comment:
I mentioned this to you elsewhere, but I think these names are too long for comfort. This doesn't need to change immediately, but I think it would be better to shorten these to just the start of the range. We have a bunch of examples of this (mostly in the HF radios which have more VFO slots than FM types). Here's an example:

https://github.com/kk7ds/chirp/blob/master/chirp/drivers/ft817.py#L236